module/BingoArea.cpp

```diff
@@ -0,0 +1,218 @@
+/**
+ * @file BingoArea.cpp
+ * @brief ビンゴエリアの情報を保持するクラス
+ * @author Hisataka-Hagiyama,uchyam
+ */
+
+#include "BingoArea.h"
+BingoArea::BingoArea()
+{
+  int index = 0, index2 = 0;  //交点サークルの番号、ブロックサークルの番号
+  //交点サークルの座標、サークルの色、黒線の本数をセット
+  for(int i = 0; i < 7; i += 2) {
+    for(int j = 0; j < 7; j += 2) {
+      COLOR initCircleColor = COLOR::NONE;
+      int initEfgeNumber = 0;

```

Comment:
```suggestion
      std::array<std::array<COLOR, 4>, 4> initCircleColor = {{COLOR::RED, COLOR::RED, COLOR::BLUE, COLOR::BLUE},
                                                             {COLOR::RED, COLOR::RED, COLOR::BLUE, COLOR::BLUE},
                                                             {COLOR::YELLOW, COLOR::YELLOW, COLOR::GREEN COLOR::GREEN},
                                                             {COLOR::YELLOW, COLOR::YELLOW, COLOR::GREEN COLOR::GREEN}};
      std::array<std::array<int, 4>, 4> initEdgeNumber = {{2, 3, 3, 2},
                                                          {3, 4, 4, 3},
                                                          {3, 4, 4, 3},
                                                          {2, 3, 3, 2}};
```
みたいにしたら，`crossCircle[index].setCicleColor(initCircleColor[i][j]);`ってするだけでセットできないかな
この方がif文が消えて視覚的にも処理的にもわかりやすくなると思うんだけどどうですか